test-runner.sh

```diff
@@ -4,6 +4,7 @@ set -o pipefail
 
 get_changes ()
 {
+	git remote remove current && \
```

Comment:
Welp really? Shit. I suppose that's cheating to just merge this in then. I'll look into getting this working right.